src/app/views/interactives/InteractivesIndex.test.js

```diff
@@ -22,8 +53,8 @@ describe("Collections", () => {
             expect(screen.getByLabelText("Internal ID")).toBeInTheDocument();
             expect(screen.getByLabelText("Title")).toBeInTheDocument();
             expect(screen.getByText("Interactive type")).toBeInTheDocument();
-            expect(screen.getByLabelText("Primary topic")).toBeInTheDocument();
-            expect(screen.getByLabelText("Data source")).toBeInTheDocument();
+            // expect(screen.getByLabelText("Primary topic")).toBeInTheDocument();
```

Comment:
this test has commented content, what is it testing?

Reply:
a new filters by type feature but the backend isn't ready for it yet so commented it out for now...